### DISCUSSION TITLE
Flutter: Widget class naming convention

### DISCUSSION THREAD

**Discussion by ptrbrynt** [netsells/code-standards#459] [Ideas]
Suggest that all widget classes should have names in the format `[Subject][Widget]`.

Good:
* `LoginWidget`
* `HomeWidget`

Bad:
* `Login`
* `LoginPage`
* `LoginScreen`

**Comment by jackcolley** [comment]
Closing / Locking as this has been added https://github.com/netsells/code-standards/blob/master/src/flutter/naming-widgets.md
